keyboards/crkbd/keymaps/like_jis/keymap.c

```diff
@@ -0,0 +1,234 @@
+#include "crkbd.h"
+#include "bootloader.h"
+#include "action_layer.h"
+#include "action_util.h"
+#include "eeconfig.h"
+#ifdef PROTOCOL_LUFA
+#include "lufa.h"
+#include "split_util.h"
+#endif
+#include "LUFA/Drivers/Peripheral/TWI.h"
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+
+#include "../lib/mode_icon_reader.c"
```

Comment:
You should probably call these via the rules.mk file, using `SRC += ...` (replacing the ... with each of the files).  But that is optional here.

Reply:
Okay, it's probably something more complicated to implement then.   

No worries, then.

Reply:
Thanks